### PR TITLE
Powerful burn damage weapons will now royally fuck you up

### PR DESCRIPTION
**Pull request by Thunder12345**
:cl: Thunder12345
tweak: Burning off limbs now has a seperate message to cutting them off
add: When your limbs are burned off, any organs in them will be incinerated
/:cl:

### CODE REVIEW

File: code/modules/surgery/bodyparts/dismemberment.dm [hunk 1]
```diff
@@ -19,8 +19,17 @@
 
 	var/obj/item/bodypart/affecting = C.get_bodypart("chest")
 	affecting.receive_damage(Clamp(brute_dam/2, 15, 50), Clamp(burn_dam/2, 0, 50)) //Damage the chest based on limb's existing damage
-	C.visible_message("<span class='danger'><B>[C]'s [src.name] has been violently dismembered!</B></span>")
+	if(dam_type == BURN)
+		C.visible_message("<span class='danger'><B>[C]'s [src.name] has been burned to ash!</B></span>")
+	else
+		C.visible_message("<span class='danger'><B>[C]'s [src.name] has been violently dismembered!</B></span>")
 	C.emote("scream")
+	if(dam_type == BURN)
```
Comment (RemieRichards) [inline code/modules/surgery/bodyparts/dismemberment.dm]: don't do this twice.

**Comment by coiax** [comment]
This will (barring podcloning) destroy the brain of anyone you behead.

**Comment by WJohn** [comment]
I can't think of any sharp flaming melee weapons aside from MAYBE the welder. Everything else is brute. Heck there are hardly any flaming melee weapons at all.

**Comment by ExcessiveUseOfCobblestone** [comment]
looks like that's a good thing considering this pr

**Comment by lzimann** [comment]
get out of here with your 1000º knifes

**Comment by Thunder12345** [comment]
@coiax intentional. The rest of your brain is cinders why isn't your brain?

**Comment by Militaires** [comment]
@lzimann NEW 1MILLIONº KNIFE VS -1MILLIONº KNIFE GONE WRONG GONE WILD

**Comment by KorPhaeron** [comment]
Hand of God

Lava Staff

Cake hat

Can all do heat dismemberment

Not sure if they should destroy brains, but considering how easy it is to get rid of brains, it probably doesn't matter

**Comment by Arianya** [comment]
Whats the dismember chance on hand of god?

And why is hand of god "sharp"?

**Comment by SgtxRootbeer** [comment]
Fe can finally make the e-sword do burn instead of brute.

**Comment by KorPhaeron** [comment]
None of those weapons are sharp, burn dismemberment is just based on weight class + damage

Dismemberment relies on reaching a damage threshold before you get a chance, so it's 3-4 hits to a limb for hand of God, and like 13-14 to the head to remove it (they are already obviously dead by the time you ever reach decapitation)

**Comment by Fox-McCloud** [comment]
Pretty iffy on this; I got to witness burn based dismemberment on Para for a long long time, and it generally wasn't very fun, at all, for players to deal wtih (though our system was...worse---laser guns could dismember in this manner).

**Comment by Kromgar** [comment]
Didnt lavastaff already do this?

**Comment by Thunder12345** [comment]
No now it burns organs too

**Comment by PKPenguin321** [comment]
Insane cakehat buff, as if it wasn't already the best murder/dismember tool now it straight up gibs too hahaha
Can't wait for this

**Comment by kevinz000** [comment]
overpowered :honk:

**Comment by SweaterKittens** [comment]
For the record, the cakehat's continued ascension as a robust weapons needs to be shot down. I want a nerf that makes it so trying to use it as a weapon sets you on fire and tells you how stupid you are for trying to use a flaming hat as a weapon.

**Comment by PKPenguin321** [comment]
Nah I love the cake hat

We need more weapons like it

**Comment by Kromgar** [comment]
So we can destroy lings with this or no?

**Comment by PKPenguin321** [comment]
Lings don't need a head/brain to live

**Comment by jughu** [comment]
@pkpenguin321 so i killed somebody i wanted to soul shard with a cakehat but i dismemberes his head off ejecting the brain making me unable get his soul

**Comment by PKPenguin321** [comment]
Congrats

**Comment by TheDreamweaver** [comment]
haha
ha...

**Comment by optimumtact** [comment]
Seems too strong sorry

**Comment by Thunder12345** [comment]
@optimumtact Too strong how? The only difference to killing someone forever is you don't have to take 2 seconds to pick up the brain afterwards.

**Comment by jughu** [comment]
@thunder12345 i dont want to accidently destroy their face with a cakehat so i cant soul shard them

**Comment by Thunder12345** [comment]
@jughu then don't hit them in the face maybe?

**Pull request closed without merging**